packages/go/openapi/src/openapi.yaml

```diff
@@ -324,6 +324,8 @@ paths:
     $ref: './paths/cypher.saved-queries.yaml'
   /api/v2/saved-queries/{saved_query_id}:
     $ref: './paths/cypher.saved-queries.id.yaml'
+  /api/v2/saved-queries/{saved_query_id}/share:

```

Comment:
We're changing to `/permissions` for these endpoints in order to point to a recognizable resource rather than having the endpoint be an action
IE:
`/share` becomes `PUT /permissions`
`/unshare` becomes `DELETE /permissions`

Reply:
Just made the changes!